core/src/alpenglow_consensus/voting_loop.rs

```diff
@@ -317,6 +320,16 @@ impl VotingLoop {
                     skip_timer.elapsed().as_millis(), timeout.as_millis()
                 );
 
+                // If we're not the leader for this slot, we should adjust the poh
+                // tick height to the beginning of this slot to keep leader detection
+                // logic accurate
+                if !is_leader {
+                    poh_recorder
+                        .write()
+                        .unwrap()
+                        .set_tick_height_to_start_of_slot(current_slot);
```

Comment:
don't know how accurate this current slot will be for transaction sending purposes, I think nozomi and triton use some sort of shred based approach?